node/src/main/scala/com/wavesplatform/transaction/DataTransaction.scala

```diff
@@ -40,8 +40,8 @@ case class DataTransaction(
 object DataTransaction extends TransactionParser {
   type TransactionT = DataTransaction
 
-  val MaxBytes: Int      = 150 * 1024 // implicitly used for RIDE CONST_STRING and CONST_BYTESTR
-  val MaxProtoBytes: Int = 165890
+  val MaxBytes: Int      = 150 * 1024 // uses for RIDE CONST_STRING and CONST_BYTESTR
+  val MaxProtoBytes: Int = 165890     // uses for RIDE CONST_BYTESTR
```

Comment:
for data or whole body? Body can have size up to 165996

Reply:
This constant is used for special protoDataPayload 
https://github.com/wavesplatform/Waves/blob/master/node/src/main/scala/com/wavesplatform/transaction/validation/impl/DataTxValidator.scala#L27